sw/host/opentitanlib/src/transport/hyperdebug/mod.rs

```diff
@@ -732,7 +734,7 @@ impl Flavor for ChipWhispererFlavor {
 
         // First, try to establish a connection to the native Chip Whisperer interface
         // which we will use for bitstream loading.
-        let board = ChipWhisperer::<Cw310>::new(None, None, None, &[])?;
+        let board = ChipWhisperer::<B>::new(None, None, None, &[])?;
```

Comment:
This is actually interesting.  In PR #19736 I am starting the work of making the `ChipWhisperer` transport not a Rust generic, but instead rely on `TransportWrapper` on top for doing all the remapping based on json files.  However, this in an instance where we use a ChipWhisperer instance without its own TransportWrapper.  No action necessary here, but I have to think about where the declarations of how the Sam3X should be used would come from (or maybe the bitstream loading is exactly the same between the two variants, if we are lucky.)

Reply:
Yes, the bitstream loading is exactly the same in both boards, only the usb productId changes.